src/laws/Laws.jl

```diff
@@ -1,27 +1,46 @@
 import Sleipnir: get_input, default_name
 
-export iTemp, iH̄, i∇S, iCPDD, iTopoRough
+export iAvgTemp, iTemp, iH̄, i∇S, iCPDD, iTopoRough
 export ConstantA, CuffeyPaterson, SyntheticC
 
 ########################
 ###### LAW INPUTS ######
 ########################
 
+"""
+    iAvgTemp <: AbstractInput
+
+Input that represents the long term air temperature over the whole glacier.
+It is computed using the OGGM climate data over a period predefined in Gungnir (i.e. around 30 years).
+"""
+struct iAvgTemp <: AbstractInput end
+
+default_name(::iAvgTemp) = :averaged_long_term_temperature
+function get_input(temp::iAvgTemp, simulation, glacier_idx, t)
+    glacier = simulation.glaciers[glacier_idx]
+    return mean(glacier.climate.longterm_temps_scalar)
+end
+function Base.zero(temp::iAvgTemp, simulation, glacier_idx)
+    glacier = simulation.glaciers[glacier_idx]
+    return zero(glacier.climate.longterm_temps_scalar)
+end
+
 """
     iTemp <: AbstractInput
 
-Input that represents the long term air temperature of a glacier.
+Input that represents the long term air temperature over the the glacier grid.
```

Comment:
I think it would be better to also name the gridded version "avg", since it's also an average long-term temperature. Otherwise, it might seem that one is the actual temperature (with its daily variability) and the other one is its average. As discussed on Slack, I'd rather rename those to `iAvgScalarTemp` and `iAvgGriddedTemp`. It's a bit long, but well. If we want to have shorter names then we cannot dispatch by type :)